docs-chef-io/content/inspec/resources/k8s_pod.md

```diff
@@ -12,57 +12,57 @@ parent = "inspec/resources/k8s"
 +++
 
 
-Use the `k8s_pod` Chef InSpec audit resource to test the configuration of specific Pods in a namespace
+Use the `k8s_pod` Chef InSpec audit resource to test the configuration of a specific Pod in the specified namespace.
 
 ## Installation
 
 ## Syntax
 
 ```ruby
-describe k8s_pods(namespace: "NAMESPACE", name: "NAME") do
+describe k8s_pod(namespace: "NAMESPACE", name: "NAME") do
   #...
 end
 ```
 
 ## Parameters
 
 `type`
-: type of the K8s resource that is for query
+: Type of the K8s resource that is for a query.
```

Comment:
This type property should be removed from here. This is not required.

Reply:
ok